chef/cookbooks/crowbar/recipes/stop-services-before-upgrade.rb

```diff
@@ -63,6 +63,7 @@
       fi
     done
   EOF
+  not_if { ::File.exist?("/usr/sbin/crm") }
```

Comment:
These are the checks we used before to differentiate HA and non-HA nodes. Do you have a better suggestion?